src/components/NewLocationPage/ChartFooter/utils.ts

```diff
@@ -34,7 +34,7 @@ export function getAddedMetricStatusText(
     ].includes(metric)
   ) {
     const hsaFormattedValue = formattedHsaCovidUsage(metric, projections);
-    return `Over the last week, the ${projections.primary.hsaName} health service area has reported having ${hsaFormattedValue} ${exploreMetricToFooterContentMap[metric].statusTextMeasure}, for an estimated ${value} coming from ${region.shortName}.`;
+    return `Over the last week, the ${projections.primary.hsaName} Health Service Area has reported having ${hsaFormattedValue} ${exploreMetricToFooterContentMap[metric].statusTextMeasure}, for an estimated ${value} coming from ${region.shortName}.`;
```

Comment:
FYI- I think Chelsi just tweaked this so you'll have to merge with her change. https://github.com/covid-projections/covid-projections/pull/5542